gatorgrouper/tests/test_arguments.py

```diff
@@ -13,7 +13,9 @@ def test_parse_arguments1():
     assert parsed_args.logging_level == logging.ERROR
     assert parsed_args.group_size == defaults.DEFAULT_GRPSIZE
     assert parsed_args.students_file == defaults.DEFAULT_CSVFILE
-    assert (parsed_args.grouping_method == group_random) is False
+    # The following line may need to be changed
+    # Unsure of the group_creation references
+    assert (parsed_args.grouping_method == group_creation) is False
```

Comment:
This does not make sense now and didn't before -- `grouping_method` is a string, and should probably reference a constant in `constants.py`.

Reply:
In addition to @Michionlion's comment, a suggestion about the syntax: `var1 != var2 ` is generally easier to read than `(var1 == var2) is False`